flink-table/flink-table-planner/src/test/java/org/apache/flink/connector/file/table/FileSystemTableSourceStreamingITCase.java

```diff
@@ -88,4 +88,61 @@ public void testMonitorContinuously() throws Exception {
 
         assertThat(actual).containsExactlyInAnyOrder(1, 2, 3, 4, 5, 6);
     }
+
+    @Test
+    public void testRegexPath() throws Exception {
```

Comment:
I think the test can't verify the `source.regex-pattern` take effect or not since it won't filer out any files.